dev-lang/vala/vala-0.36.12.recipe

```diff
@@ -4,20 +4,24 @@ programming language features to GNOME developers without imposing any \
 additional runtime requirements and without using a different ABI \
 compared to applications and libraries written in C."
 HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
-COPYRIGHT="2006-2016 Jürg Billeter
-	2006-2016 Raffaele Sandrini"
+COPYRIGHT="2006-2018 Jürg Billeter
+	2006-2018 Raffaele Sandrini"
 LICENSE="GNU LGPL v2.1"
 REVISION="1"
 SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
-CHECKSUM_SHA256="e9f23ce711c1a72ce664d10946fbc5953f01b0b7f2a3562e7a01e362d86de059"
+CHECKSUM_SHA256="b4081d3d313cf9c3e8d7d795983cf8f7ed99286d1abc1b5709b0f17883387cdb"
 SOURCE_DIR="vala-$portVersion"
 PATCHES="vala-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
 
```

Comment:
You keep comming up with variables ... iirc I didn't call for portVersionCompat in the first place ...

Reply:
I'm suggesting `portVers` because there are a lot of occurences of `"${portVersion%.*}"`.
I think the suggestion makes the recipe more eye-friendly, but this is just my opinion.
As indicated, this is just a suggestion. So please feel free to do whatever you wish.

Reply:
I don't see a benefit ... you provide the same ...